### ISSUE TITLE
Add tests for `yarn check`

### ISSUE THREAD

**Issue by voxsim** [yarnpkg/yarn#3893]
`yarn check` has not tests at all and we should at least cover:
* Do a check without `verifyTree` and `integrity` flags
* Do a check with `verifyTree` flag
* Do a check with `integrity` flag

**Comment by peterschussheim** [comment]
@voxsim Can I take this one?

**Comment by voxsim** [comment]
@peterschussheim yep go ahead!

**Comment by peterschussheim** [comment]
Great!  Are there any examples you know would be a good starting point?

**Comment by voxsim** [comment]
@peterschussheim nope, you can take inspiration from the other tests in the test forlder (for example https://github.com/yarnpkg/yarn/blob/master/__tests__/commands/licenses.js)

**Comment by rubycut** [comment]
@BYK , owner command seems to be broken, it doesn't return anything. I am trying to fix it, but I am now getting 1 maintainer @Daniel15 , but if I try to  run ```npm owner list yarn```, I get six maintainers.
